tools/transform.cpp

```diff
@@ -419,6 +465,13 @@ static void calculateAndInitConstants(Transform &t) {
           has_int2ptr |= conv->getOp() == ConversionOp::Int2Ptr;
           has_ptr2int |= conv->getOp() == ConversionOp::Ptr2Int;
         }
+
+        auto accsz = get_access_size(I);
+        if (accsz != NO_ACCESS){
+          min_access_size = accsz == UNKNOWN ? 1 :
+              (min_access_size == UNKNOWN ? accsz :
+                                            gcd(min_access_size, accsz));
```

Comment:
to simplify this, you could set UNKNOWN to 1, since that's the alignment when unknown, and set min_access_size to some max alignment, say 512. GCD will take care of bringing that down (it's all powers of 2, so it's fine).